Tests/UnitTests/CrowdinStringTests.swift

```diff
@@ -30,7 +30,7 @@ class CrowdinStringTestsLocalization: XCTestCase {
     }
     
     func testCrowdinSDKStringLocalizationForDefaultLanguage() {
-        CrowdinSDK.enableSDKLocalization(true, localization: nil)
+        CrowdinSDK.enableSDKLocalization(true, localization: "en")
```

Comment:
Why it's changed?

Reply:
Unit test failed otherwise- maybe you have some clue why it's happen? Thx